### PR TITLE
doc: add minutes for meeting May 3 2023

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2023-05-03.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Board updates
+  * CPC proposing some changes to foundation bylaws in respect to
+  * how community directors are elected. Important that people
+  * take a look to make sure they don’t have any concerns.
+  * <https://github.com/openjs-foundation/cross-project-council/pull/1055>
```
Comment (MoLow) [inline meetings/2023-05-03.md:31]:
```suggestion
  CPC proposing some changes to foundation bylaws in respect to
  how community directors are elected. Important that people
  take a look to make sure they don’t have any concerns.
  <https://github.com/openjs-foundation/cross-project-council/pull/1055>
```

File: meetings/2023-05-03.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### nodejs/TSC
+
+* Adding the OpenJS Foundation staff as a billing manager for nodejs-private [#1376](https://github.com/nodejs/TSC/issues/1376)
+* No objections from the 6 TSC members at the meeting, will make note in issue and the remove from agenda.
```
Comment (MoLow) [inline meetings/2023-05-03.md:36]:
```suggestion
  * No objections from the 6 TSC members at the meeting, will make note in issue and the remove from agenda.
```

File: meetings/2023-05-03.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* openjs-foundation/cross-project-council [#397](https://github.com/openjs-foundation/cross-project-council/issues/397)
+  * Travel fund was migrated from Node.js project to CPC/Foundation level so all projects could use the travel fund
+  * Has enabled lots of things
+  * Today you must make public request with info on conference info, travel dates etc.
+  * Question being discussed about privacy. The requesting info might be problematic.
+  * Discussion is that requests may be made private versus public
+  * If anybody has concerns about requests being private versus public please jump into the discussion.
+  * Danielle, have not asked for funds due to public nature of it so has been a barrier in that sense.
+
+## Strategic Initiatives
```
Comment (RaisinTen) [inline meetings/2023-05-03.md:53]:
```suggestion
## Strategic Initiatives

* ShadowRealm (Chengzhong) - Working on the finalization order of base objects and their nested fields when the shadow realm is being finalized - <https://github.com/nodejs/node/pull/47688>.
```

from @legendecas's comment in https://github.com/nodejs/TSC/issues/1381#issuecomment-1532988149

**Review by MoLow — APPROVED** [review]

File: meetings/2023-05-03.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Board updates
+  CPC proposing some changes to foundation bylaws in respect to
+  how community directors are elected. Important that people
+  take a look to make sure they don’t have any concerns.
+  <https://github.com/openjs-foundation/cross-project-council/pull/1055>
```
Comment (tniessen) [inline meetings/2023-05-03.md:31]:
```suggestion
  * CPC proposing some changes to foundation bylaws in respect to
    how community directors are elected. Important that people
    take a look to make sure they don’t have any concerns.
    <https://github.com/openjs-foundation/cross-project-council/pull/1055>
```

**Review by RaisinTen — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Pull request merged**